play-services-base/src/main/java/com/google/android/gms/auth/api/signin/GoogleSignInOptions.java

```diff
@@ -233,6 +233,17 @@ public Builder requestScopes(@NonNull Scope scope, @NonNull Scope... scopes) {
             return this;
         }
 
+        /**
+         * Specifies OAuth 2.0 scopes your application requests. See {@link Scopes} for more information.
+         *
+         * @param scopes More OAuth 2.0 scopes requested by your app.
+         */
+        @NonNull
+        public Builder requestScopes(@NonNull List<Scope> scopes) {
+            this.scopes.addAll(scopes);
+            return this;
+        }
+
```

Comment:
This is not part of the public API. If it is really needed, and `@Hide`, but better consider using the existing `requestScopes()` method.